### ISSUE TITLE
New User Notifications wording

### ISSUE THREAD

**Issue by misulicus** [pippinsplugins/Restrict-Content-Pro#208]
I looked at the code and aparently the description of the option might be confusing.
The checkbox 'disable_new_user_notices' when enabled it will no longer notify the user on any of the membership statuses (cancel/expired/etc)
Might need to maybe rephrase?

**Comment by pippinsplugins** [comment]
This is just to disable the site admin notifications, not the customer notifications.

It says this right now: `Check this option if you do NOT want to receive emails when new users signup.`

I think that's correct.
